README.md

```diff
@@ -90,7 +90,7 @@ CIP Editors meetings are public, recorded, and [published on Youtube](https://ww
 | 0088 | [Token Policy Registration](./CIP-0088) | Proposed |
 | 0093 | [Authenticated Web3 HTTP requests](./CIP-0093) | Proposed |
 | 0094 | [SPO On-chain Polls](./CIP-0094) | Active |
-| 0095 | [Web-Wallet Bridge - Governance](./CIP-0095) | Proposed |
+| 0095 | [Web-Wallet Bridge - Conway](./CIP-0095) | Active |
```

Comment:
```suggestion
| 0095 | [Web-Wallet Bridge - Conway Era](./CIP-0095) | Active |
```

Reply:
It may be of use going forward that CIP-XXXX would define this as the _Conway **ledger** era_ (Conway = capitalised as a unique proper noun, while "ledger era" is a common noun phrase that is not unique, so lowercase).

Reply:
that is a **great** suggestion Robert, I had forgotten about CIP59!
I will edit the above to reflect this!

Reply:
... and thanks for finishing the other half of the change 🤪